pubs/collab-patterns/collab.Rmd

```diff
@@ -26,9 +26,173 @@ output:
 ```{r code = readLines("../load_data.R"), echo = F, message = F}
 ```
 
+Rewritting some of the shared_utils functions to be used in R.
```

Comment:
1. Please move these functions to the end of the setup bloc.
2. Why is there a newline after every line?
3. Please briefly comment each function to explain what/how it works. See examples in load_data.R, at the bottom.